schema/properties_directory/structural/basis/atomic_coordinates.json

```diff
@@ -4,16 +4,16 @@
     "title": "atomic coordinates",
     "description": "coordinates of atoms by ids, vector, unitless",
     "properties": {
-        "name": {
-            "enum": [
-                "atomic_coordinates"
-            ]
+        "id": {
```

Comment:
I think we should rename the files to `atomic_coordinate.json` and `atomic_element.json` as to not create confusion about whether it is an array or not.